sql-cli/sql_cli/__main__.py

```diff
@@ -6,28 +6,45 @@
 from rich import print as rprint
 
 import sql_cli
+from sql_cli.astro.command import AstroCommand
+from sql_cli.astro.group import AstroGroup
 from sql_cli.constants import DEFAULT_AIRFLOW_HOME, DEFAULT_DAGS_FOLDER
 
 load_dotenv()
-app = typer.Typer(add_completion=False, context_settings={"help_option_names": ["-h", "--help"]})
+app = typer.Typer(
+    name="flow",
+    cls=AstroGroup,
+    add_completion=False,
+    context_settings={"help_option_names": ["-h", "--help"]},
+    rich_markup_mode="rich",
```

Comment:
This solves the issue below with the `\b\n` for new lines in the help output + we can use rich formatting in the help if needed.

Reply:
[Look](https://typer.tiangolo.com/tutorial/commands/help/#rich-markdown-and-markup) how beautiful it could look like 🤩

Reply:
This looks amazing, @feluelle ! :clap: :clap: